src/goose.rs

```diff
@@ -2318,6 +2318,7 @@ pub(crate) fn create_reqwest_client(
         .timeout(Duration::from_millis(timeout))
         // Enable gzip unless `--no-gzip` flag is enabled.
         .gzip(!configuration.no_gzip)
+        .danger_accept_invalid_certs(configuration.accept_invalid_certs)
```

Comment:
Please include a comment, for example:

`// Validate https certificates unless `--accept_invalid_certs` is enabled.`

Reply:
Whoops, sorry, I think that's actually `--accept-invalid-certs` -- can you test and confirm, and fix if necessary?

Reply:
Thanks!